frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/multiple-pool.ts

```diff
@@ -0,0 +1,73 @@
+import { NS } from '../utils/consts';
+
+// Storage class form
+export const createStorageClassButton: string = 'item-create';
+
+// Create storage class from
+export const provisionerDropdown: string = 'storage-class-provisioner-dropdown';
+export const expandPvcCheckbox: string = 'expand-pvc-checkbox';
+export const storagePoolDropdown: string = 'pool-dropdown-toggle';
+export const createNewPoolButton: string = 'create-new-pool-button';
+
+// Create new pool form
+export const titleLabel: string = 'modal-title';
+export const poolNameTextBox: string = 'new-pool-name-textbox';
+export const replicaDropdown: string = 'replica-dropdown';
+export const confirmAction: string = 'confirm-action';
+
+// Pool status
+export const emptyStateBody: string = 'empty-state-body';
+export const poolStatus: string = 'Ready';
```

Comment:
The name of the variable doesn't truly reflect what it does. It would be better if it were an `enum`. 
```suggestion
export enum PoolStatus {
  READY = 'Ready'
  }
```